postman/schemas/transfer.yml

```diff
@@ -593,6 +593,212 @@ paths:
             application/json:
               schema:
                 "$ref": "#/components/schemas/response401"
+  "/quotes":
+    get:
+      tags:
+      - Helpers
+      summary: Get Quotes
+      description: |
+        Given a corridor and a sender amount, this operation returns
+        quotes which include the exchange rate, fee adjustments, disclosures, and
+        recipient amount.<br>
+        The app can present the
+        quotes to the end-user who can decide whether to proceed with the transfer.<br>
+        Quotes are accurate at the instance of the quote, but rates may change between
+        the time of the quote and the time of the transfer. The Execute Transfer operation
+        returns the final exchange rate.<br><br>
+        __Notes__:
+        - See [Currency](https://developer.readyremit.com/reference/about-the-reference#currency)
+        for more about currency values and decimal places.
+        - If __transferMethod__ is not supplied, quotes will be generated for the corridor's available transfer methods.
+        - The __id__ returned for a specific quote will need to be supplied to the Execute Transfer operation.  It will be assigned to __quoteHistoryId__ in the Body Params. 
+        - The __200 response__ could contain both quotes and errors if no __transferMethod__ is supplied since it may have attempted to generate more than one quote.
+
+        
+      operationId: getQuotes
+      parameters:
+      - name: dstCountryIso3Code
+        in: query
+        description: See [country codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3).
+        required: true
+        style: form
+        explode: true
+        schema:
+          "$ref": "#/components/schemas/threeUppercaseChars"
+      - name: dstCurrencyIso3Code
+        in: query
+        description: See [currency codes](https://en.wikipedia.org/wiki/ISO_4217).
+        required: true
+        style: form
+        explode: true
+        schema:
+          "$ref": "#/components/schemas/threeUppercaseChars"
+      - name: srcCurrencyIso3Code
+        in: query
+        description: See [currency codes](https://en.wikipedia.org/wiki/ISO_4217).
+        required: true
+        style: form
+        explode: true
+        schema:
+          "$ref": "#/components/schemas/threeUppercaseChars"
+      - name: transferMethod
+        in: query
+        required: false
+        style: form
+        explode: true
+        schema:
+          "$ref": "#/components/schemas/transferMethodsEnum"
+      - name: quoteBy
+        in: query
+        required: true
+        style: form
+        explode: true
+        schema:
+          "$ref": "#/components/schemas/quoteBy"
+      - name: amount
```

Comment:
The amount field in other API calls has been a little inconsistent. Sometimes we ask them for the amount with decimal places. Sometimes we ask for the "decimal normalized" amount (an integer with the decimals multiplied out). I'd recommend clarity on how this field should be formatted

Reply:
Changed the type and added more to the description.